packages/block-library/src/page-list/edit.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+
+import { useBlockProps } from '@wordpress/block-editor';
+import ServerSideRender from '@wordpress/server-side-render';
+import { useSelect } from '@wordpress/data';
+
+export default function PageListEdit( { clientId } ) {
+	const navigationBlockAttributes = useSelect( ( select ) => {
+		const { getBlockAttributes, getBlockParentsByBlockName } = select(
+			'core/block-editor'
+		);
+		const parentBlock = getBlockParentsByBlockName(
+			clientId,
+			'core/navigation'
+		)[ 0 ];
+		return getBlockAttributes( parentBlock );
```

Comment:
I think we should be able to use block context instead of selecting from the navigation block's attributes via the block-editor store.

The good thing is it's a bit easier, it's just passed in as a prop:
https://developer.wordpress.org/block-editor/developers/block-api/block-context/#using-block-context

Looks like we should also make this change to nav link at some point as well.